### ISSUE TITLE
VB -> C#: statics in methods with escaped names get invalid names

### ISSUE THREAD

**Issue by TymurGubayev** [icsharpcode/CodeConverter#1043]
### VB.Net input code
```vbnet
Public Class KeywordName
    Sub [Step]()
        Static i As Integer
    End Sub
End Class
```

### Erroneous output
```C#
    public class KeywordName
    {
        private int _[Step]_i;
        public void Step()
        {
        }
    }
```

### Expected output
```C#
    public class KeywordName
    {
        private int _Step_i;
        public void Step()
        {
        }
    }
```

### Details
* Product in use: VS extension
* Version in use: 2e3da7eb0abad1fddd4976a7a8735e3d926b84ff
* Did you see it working in a previous version, which? No
* Any other relevant information to the issue, or your interest in contributing a fix.

**Comment by GrahamTheCoder** [comment]
Same area of code as https://github.com/icsharpcode/CodeConverter/issues/1044

**Comment by TymurGubayev** [comment]
the issue is still there, the expected code in the test is incorrect

**Comment by GrahamTheCoder** [comment]
New attempted fix in 825cc5e1235bce46416921c402d9a96ee596bb76
